packages/orchestrator-ui-components/src/rtk/endpoints/streamMessages.ts

```diff
@@ -23,11 +24,9 @@ const getWebSocket = async (url: string) => {
     }
 };
 
-// Recommended values are between 30 and 60 seconds
-// Note: When the browser tab is suspended, the browser will give this suspended tab approximately every 60 seconds time
-// to execute code. This means that the ping message will be sent every 60 seconds in suspended mode. This depends
-// on the browser and can differ.
-const PING_INTERVAL_MS = 45000;
+const PING_INTERVAL_MS = 30000;
+const NO_PONG_RECEIVED_INTERVAL_MS = 35000;
```

Comment:
Maybe `NO_PONG_RECEIVED_TIMEOUT`? (MS is always a given right?)

Could also introduce a `const SECONDS=1000` so you get `35 * SECONDS` as value

Reply:
Used the TIMEOUT suggestion. I do prefer MS in the name so didn't change that